docs/packages.rst

```diff
@@ -0,0 +1,92 @@
+Via Package Manager
+===================
+
+You can install xonsh using ``conda``, ``pip`` or the package manager for 
+your operating system distribution.
+
+For the fullest interactive user experience, these additional packages should also be installed:
+
+  :prompt-toolkit: for command completion, configurable key bindings and especially multi-line line editing.
+  :pygments: for xonsh and python syntax-specific highlighting
+  :setproctitle: updates process title (in terminal window and process monitor) to match Xonsh arguments.
+
+Installing with these packages is the recommended configuration and is documented first.   
+If you are operating in a specialized or restricted environment, you can install just the xonsh package, as 
+described in `fewer prerequisites`_
+
+
+**conda:**
+
+.. code-block:: console
+
+    $ conda config --add channels conda-forge
+    $ conda install pygments prompt-toolkit setproctitle xonsh
+
+
+**pip:** Typically you will activate a virtual environment and install xonsh there.  This will ensure that you invoke the 
+correct python interpreter and ``pip`` module. 
+
+.. code-block:: console
+
+    $ pip install pygments prompt-toolkit setproctitle xonsh
+
+As a shortcut (for pip only), you can specify all of the above with `extras` syntax:
+
+.. code-block:: console
+
+    $ pip install xonsh[full]
+
+The above ``pip`` commands may have to be spelled ``pip3`` or ``sudo pip3`` if you are not installing in a virtual environment.
+
+**source:** Pip can also install the most recent xonsh source code from the 
+`xonsh project repository <https://github.com/xonsh/xonsh>`_.
+
+.. code-block:: console
+
+    $ pip install pygments prompt-toolkit setproctitle https://github.com/xonsh/xonsh/archive/master.zip
```

Comment:
I think we should recommend people clone the repo and then `pip install --user .`  rather than use the master archive

Reply:
But that would not be a source install?  I admit I am fuzzy on the difference, or the tradeoffs...